### PR TITLE
16.0 mig account fiscal year auto create (from 12.0)

### PR DESCRIPTION
**Pull request by legalsylvain**
trivial migration

    - depends on new OCA module account_fiscal_year
    - bump version
    - use new python convention

-

### CODE REVIEW

File: account_fiscal_year_auto_create/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Once installed, a cron is running each day. It will create, for each company,
+a new fiscal year, if it is the last day of the current fiscal year.
+
+This module is interesting specially in multi company context, to avoid annoying setup every year.
+
+The fiscal year created has a classical "12 monthes" duration, but the accountant
+can modify it, once created.
+
+Exemple
```
Comment (bealdav) [inline account_fiscal_year_auto_create/readme/DESCRIPTION.rst]: Nitpicking not mandatory

```suggestion
Example
```

**Review by rafaelbn — APPROVED** [review]
👍 

@bealdav ? What do you think? 😄

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**